packages/schema/src/camelCase.ts

```diff
@@ -71,13 +71,10 @@ export class CamelCase extends PrismaReader {
     }
   }
 
-  private convertWord(word: string, model = false) {
```

Comment:
and here you removed the arg that make me know If I work with the model name or the field and with current code, you will update all fields to start with upper char